### PR TITLE
[Core][All]Consistency in C++ files (GetDefaultSettings->GetDefaultParameters)

### PR DESCRIPTION
**Pull request by loumalouomega**
**Description**
Part of #7446. Changes the GetDefaultSettings to GetDefaultParameters

**Changelog**
- Refactor some conflicting static methods in order to compile and work
- Changes the GetDefaultSettings to GetDefaultParameters

### CODE REVIEW

**Review by rubenzorrilla — APPROVED** [review]

File: kratos/solving_strategies/schemes/residualbased_incremental_aitken_static_scheme.h [hunk 1]
```diff
@@ -229,6 +218,25 @@ class ResidualBasedIncrementalAitkenStaticScheme : public ResidualBasedIncrement
         mOldOmega = Omega;
     }
 
+    static Parameters StaticGetDefaultParameters()
```
Comment (philbucher) [inline kratos/solving_strategies/schemes/residualbased_incremental_aitken_static_scheme.h:221]: why is this needed, why cannot you use the defaut  method?

**Review by philbucher — CHANGES_REQUESTED** [review]

File: kratos/solving_strategies/schemes/residualbased_incremental_aitken_static_scheme.h (hunk 1 above)
Comment (loumalouomega) [reply kratos/solving_strategies/schemes/residualbased_incremental_aitken_static_scheme.h:221]: Nope, it is used in the constructor, does not compile

**Review by philbucher — APPROVED** [review]
hmm not the cleanest solution IMO but now I understand, thx for pointing it out

@rubenzorrilla I think this is mostly your code, maybe you can clean / improve this next time you change sth

**Comment by rubenzorrilla** [comment]
> 
> 
> hmm not the cleanest solution IMO but now I understand, thx for pointing it out
> 
> @rubenzorrilla I think this is mostly your code, maybe you can clean / improve this next time you change sth

I don't think there is a better solution... the point is that I'm accessing the defaults from within the constructor. As there is no instance yet the method needs to be static. I don't think this is a bad solution. Actually, I like how @loumalouomega resolved it.
